horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/RunServiceTest.java

```diff
@@ -756,27 +756,6 @@ public void testUploadToPrivateTest() throws JsonProcessingException {
         assertEquals(test.name, response.testname);
     }
 
-    @org.junit.jupiter.api.Test
-    public void testUploadToPrivateUsingToken() {
-        final String MY_SECRET_TOKEN = "mySecretToken";
-        Test test = createExampleTest("supersecret");
-        test.access = Access.PRIVATE;
-        test = createTest(test);
-
-        // TestToken.value is not readable, therefore we can't pass it in.
-        addToken(test, TestTokenDAO.READ + TestTokenDAO.UPLOAD, MY_SECRET_TOKEN);
-
-        long now = System.currentTimeMillis();
-        RestAssured.given()
-                .header(HttpHeaders.CONTENT_TYPE, "application/json")
-                .body(org.testcontainers.shaded.com.fasterxml.jackson.databind.node.JsonNodeFactory.instance.objectNode())
-                .post("/api/run/data?start=" + now + "&stop=" + now + "&test=" + test.name + "&owner=" + UPLOADER_ROLES[0] +
-                        "&access=" + Access.PRIVATE + "&token=" + MY_SECRET_TOKEN)
-                .then()
-                .statusCode(200)
-                .extract().asString();
-    }
-
```

Comment:
Do we have an equivalent test that uploads to a private test using an API key?

Reply:
I have added an integration test.